libkbfs/interfaces.go

```diff
@@ -529,17 +530,25 @@ type mdDecryptionKeyGetter interface {
 		kbfscrypto.TLFCryptKey, error)
 }
 
-// KeyManager fetches and constructs the keys needed for KBFS file
-// operations.
-type KeyManager interface {
-	encryptionKeyGetter
-	mdDecryptionKeyGetter
-
+type blockDecryptionKeyGetter interface {
 	// GetTLFCryptKeyForBlockDecryption gets the crypt key to use
 	// for the TLF with the given metadata to decrypt the block
 	// pointed to by the given pointer.
 	GetTLFCryptKeyForBlockDecryption(ctx context.Context, kmd KeyMetadata,
 		blockPtr BlockPointer) (kbfscrypto.TLFCryptKey, error)
+}
+
+type blockKeyGetter interface {
+	encryptionKeyGetter
+	blockDecryptionKeyGetter
+}
+
+// KeyManager fetches and constructs the keys needed for KBFS file
+// operations.
+type KeyManager interface {
+	encryptionKeyGetter
```

Comment:
Could just make this `blockKeyGetter` and remove `blockDecryptionKeyGetter` below to save a line, up to you.